tools/autograph-monitor/constants.go

```diff
@@ -8,100 +8,83 @@ const autographDevRootHash = `5E:36:F2:14:DE:82:3F:8B:29:96:89:23:5F:03:41:AC:AF
 
 // firefoxPkiStageRootHash is the SHA2 hash of the firefoxPkiStageRoot
 // cert raw bytes
-const firefoxPkiStageRootHash = `DB:74:CE:58:E4:F9:D0:9E:E0:42:36:BE:6C:C5:C4:F6:6A:E7:74:7D:C0:21:42:7A:03:BC:2F:57:0C:8B:9B:90`
+const firefoxPkiStageRootHash = `C0:F0:5D:59:B1:FD:E2:57:80:85:4C:32:FA:E8:FA:BA:84:81:C2:33:B4:C1:D3:90:CC:A5:F2:CE:A8:19:30:EE`
```

Comment:
This is only one of the two XPI roots that are used in staging. You'll need to include both. The two were used to do initial "backwards compat" testing during cert succession because the prior staging root key had been lost to time.

Both are on https://bugzilla.mozilla.org/show_bug.cgi?id=1882192 (and soon https://github.com/mozilla-services/hsm/pull/31)

The cas_new is the "long-term" staging root we're using and is used for all of the `cas_new_` signers in staging of types `xpi` and `contentsignaturepki` and will be in all new signers of those types. 

cas_cur was the root that'll expire in a few months and is used for all of the `cas_cur_` signers in staging of both `xpi` and `contentsignaturepki` types.

The easiest thing to do would be to add both as constants here and then add both to the truststore and do the sha checking for both. Key agility, baby.

Reply:
Oh, also, I don't think we've removed the old staging signer ids, yet, either, so you probably ALSO need to keep the current constant in place, too.

Reply:
Wellll, that actually would be a good forcing function for us to remove the signers using that old staging root, actually.

Reply:
I've converted this to an array for now and have the 3 root certs in there.

Maybe I'm thinking about this backwards. But we should remove the certs from the store before removing them from monitor, right?

Reply:
You're correct, imo. We should see this working and then see it continue to work after we remove them from the configs.

It could go the other way, but this lets us scream test at our leisure instead of gating the GCP migration on the scream test having without screams.